firefox_ui_harness/runners/base.py

```diff
@@ -2,7 +2,16 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-default_prefs = {
+import os
+
+import mozinfo
+from marionette import BaseMarionetteTestRunner
+
+import firefox_ui_tests
+from ..testcase import FirefoxTestCase
+
+
+DEFAULT_PREFS = {
     'app.update.auto': False,
     'app.update.enabled': False,
```

Comment:
Strange. I also see this now. I don't know what was going on earlier today. At least this was the problem why my local update tests were failing yesterday.